demo/mkldnn_quant/README_en.md

```diff
@@ -0,0 +1,181 @@
+- **with_accuracy_layer:** If the infer_model require only image input or image input and label input. Default value false.
+-**optimize_fp32_model** Whether to optimize the FP32 model. The sample can test INT8 model, or optimize (fuses, etc.) and test FP32 model. The default is false, which means do not do optimization.
+
+You can directly modify MODEL_DIR and DATA_DIR in `run.sh` under `/PATH_TO_PaddleSlim/demo/mkldnn_quant/quant_aware/` directory, then you can execute `./run.sh` for CPU inference.
+
+### 4.3 Users write their own tests:
+If the user writes his own test:
+1. Test the INT8 model
+    If the user tests the converted INT8 model, use paddle::NativeConfig to test. In the demo, set `optimize_fp32_model` to false.
+2. Test the FP32 model
+   If you want to test the PF32 model, you can use AnalysisConfig to optimize the original FP32 model (fuses, etc.), it will speed up FP32 model performance. AnalysisConfig configuration are as follows:
+```
+static void SetConfig(paddle::AnalysisConfig *cfg) {
+  cfg->SetModel(FLAGS_infer_model); // Required. The model to be tested
+  cfg->DisableGpu(); // Required. Deploy on the CPU to predict, you must Disablegpu
+  cfg->EnableMKLDNN(); // Required. Configure with MKLDNN enabled make the inference faster than native configuration
+  cfg->SwitchIrOptim(); // If the original FP32 model is passed in SetModel, setting SwitchIrOptim to true will fuses many ops hence improve the performance
+  cfg->SetCpuMathLibraryNumThreads(FLAGS_num_threads); // The default setting is 1.
+}
+```
+**In the demo we provided**
+- If `infer_model` is set FP32 model and `optimize_fp32_model` is set to true, the above AnalysisConfig configuration will be executed automatically. Hence the FP32 model will be fused and optimized, the performance should be faster than FP32 inference using NativeConfig.
+- If `infer_model` is converted DNNL INT8 model, `optimize_fp32_model` will make no difference, because INT8 model has been fused, optimized and quantized.
+- If `infer_model` is set fake quantized model generated by PaddleSlim, `optimize_fp32_model` does not work even if it is set to true, because the fake quantized model contains fake quantize/dequantize ops, which cannot be fused or optimized.
+
+## 5. Accuracy and performance benchmark
+INT8 model accuracy and performance results refer to [CPU deployment predicts the accuracy and performance of INT8 model](https://github.com/PaddlePaddle/PaddleSlim/tree/develop/docs/zh_cn/tutorials/image_classification_mkldnn_quant_aware_tutorial.md)
+
+## FAQ
+
+- For deploying INT8 NLP models on CPU, refer to [ERNIE model quant INT8 accuracy and performance reproduction](https://github.com/PaddlePaddle/benchmark/tree/master/Inference/c%2B%2B /ernie/mkldnn)
+- The detailed DNNL quantification process can be viewed in [SLIM quant for INT8 DNNL](https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/fluid/contrib/slim/tests/QAT_mkldnn_int8_readme.md)
```

Comment:
the path has changed:
"The details of the DNNL quantization process are described in [SLIM quant for DNNL INT8](https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/fluid/contrib/slim/tests/README.md)."